client/src/utils/API.js

```diff
@@ -12,14 +11,13 @@ export default {
     return axios.post("/api/createjobs", jobData);
   },
 
-  // TODO CHILD & CHILDREN Section
-
-  getAddChild: function(childrenData) {
-    return axios.post("/api/addchild", childrenData);
+  createChild: function(childrenData) {
+    return axios.post("/api/createChild", childrenData);
```

Comment:
what happens when childrenData is null? what happens when this axios post doesnt return what you expect are you handling the error correctly?

Reply:
logged to console (controllers section)